doc/rbd/rbd-mirroring.rst

```diff
@@ -228,6 +228,34 @@ For example::
    local cluster's ``rbd-mirror`` daemon process is responsible for performing
    the resync asynchronously.
 
+Mirror Status
+=============
+
+For every mirroring image the ``rbd-mirror`` daemon stores its current status
+in the RBD mirror image directory. The mirroring status can be retrieved using
+the ``mirror image status`` and ``mirror pool status`` commands.
+
+To get status for an image, specify the ``mirror image status`` command along
```

Comment:
Minor (just to keep it consistent with the rest of the doc): "To request the mirror image status with ``rbd``, specify the ``mirror image status`` command along with the pool and image name::"